students/jayjohnson/lesson01/break_me.py

```diff
@@ -0,0 +1,37 @@
+'''
+Jay Johnson lesson01
+
+In the break_me.py file write four simple Python functions:
+
+Each function, when called, should cause an exception to happen
+Each function should result in one of the four most common exceptions you’ll find.
+for review: NameError, TypeError, SyntaxError, AttributeError
+(hint – the interpreter will quit when it hits a Exception
+– so you can comment out all but the one you are testing at the moment)
+
+'''
+
+# error functions
+def nameError_fuction():
+    a = 5
+    del a
+    a
+
+def typeError_fuction():
+    type = 'abc'
```

Comment:
Since the point of this exercise is getting errors, reassigning a builtin function is a good way to do that I suppose, so good work :). An example of a more common type error could be like:
```
def typerr_fun():
    return "35"/5
```